src/ActionModal/utils/ActionManager.js

```diff
@@ -76,6 +76,13 @@ export default class ActionManager {
       this.message = await this.createWithdrawTransaction()
     }
 
+    // temporary fix as the SDK doesn't return proper estimates for votes
+    // TODO move into transacton service
+    /* istanbul ignore next */
+    if (this.messageType === transaction.VOTE) {
+      gasEstimate = 30000
```

Comment:
how does this compare to what is usually returned?

Reply:
and what gas prices should this typically generate?

Reply:
I can't say for sure. On the local testnet it is 13k. I saw gas prices so far up to 25k. I guess would be cool to track used gas of all txs.
This effects fees. It makes the txs proportionally more expensive. Still txs in general are very cheap.